### PR TITLE
Restrict required behat version

### PR DESCRIPTION
**Pull request by yoanm**
Related to #6

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "require": {
         "php":                          ">=5.3.2",
 
-        "behat/behat":                  ">=2.4",
+        "behat/behat":                  "~2",
```
Comment (jakzal) [inline composer.json]: Can you use `^2.4|^3.0` please? 

`<=2.3` is not compatible with this extension. Have you verified `3.0`?

Comment (jakzal) [inline composer.json]: Readme also needs an update.

Comment (yoanm) [inline composer.json]: > Have you verified 3.0

Yep, it's not working with this version as [`Extension`](https://github.com/PolishSymfonyCommunity/Symfony2MockerExtension/blob/master/src/PSS/Behat/Symfony2MockerExtension/Extension.php#L12) use [`ExtensionInterface`](https://github.com/Behat/Behat/blob/2.5/src/Behat/Behat/Extension/ExtensionInterface.php) that was removed on Behat 3.0 !

**Comment by jakzal** [comment]
@yoanm `>=2.4` should work for `2.5` as well (actually for `3.0` too). I've create a branch for `1.0`. `master` is `2.0` now with support for Behat 3.

**Pull request closed without merging**